features/font-stretch.yml

```diff
@@ -0,0 +1,12 @@
+name: font-stretch
+description: The `font-stretch` CSS property selects a font face from a font family based on width, either by a keyword such as "condensed" or a percentage. The `font-stretch` property will be replaced by `font-width`.
```

Comment:
`font-width` exists in no implementations yet (AFAIK). Let's move that to a comment and return to this when there's something contemporary to reference.

```suggestion
description: The `font-stretch` CSS property selects a font face from a font family based on width, either by a keyword such as `condensed` or a percentage.
# Note: `font-stretch` will become an alias to `font-width`: https://drafts.csswg.org/css-fonts/#font-stretch-desc
```